README.md

```diff
@@ -8,9 +8,9 @@
 
 
 ## Build images
-2 images will be built, komodod and marketmaker:
+Build the TxBlaster Image:
 ```
-docker-compose build marketmaker
```

Comment:
Please leave building of images via docker-compose there. docker-compose.yml knows how to build/name the image.